terraform/modules/relay-app/main.tf

```diff
@@ -23,8 +23,12 @@ locals {
       value = var.observability_log_level
     },
     {
-      name  = "GOOGLE_CLOUD_PROJECT_ID"
-      value = var.project_id
+      name  = "LOG_FORMAT"
+      value = "google-cloud"
+    },
+    {
+      name  = "TRACE_RECEIVER"
```

Comment:
I think the _more_ conventional name is `COLLECTOR` but this one should be fine too.